app/src/pages/inside/stepPage/modals/makeDecisionModal/machineLearningSuggestions/machineLearningSuggestions.jsx

```diff
@@ -160,28 +159,19 @@ export const MachineLearningSuggestions = ({
                 </span>
               </div>
               <div className={cx('suggestion-item-wrapper')} key={testItemResource.id}>
-                <ItemHeader
+                <TestItemDetails
                   item={testItemResource}
                   selectItem={selectMachineLearningSuggestionItem}
                   isSelected={modalState.source.id === testItemResource.id}
                   onClickLinkEvent={onClickExternalLinkEvent}
+                  logs={logs}
+                  suggestRs={suggestRs}
+                  mode={ANALYZE_OR_SUGGEST_ITEM}
```

Comment:
Please see the [comment](https://github.com/reportportal/service-ui/pull/2904#discussion_r744995063) from @chivekrodis .
In this case you are trying to include some possible cases of usage for this item into one variable.
What if we will have another one item type that requires such view? 
From my point of view we have three item types not depending where they are used:
1. Radio
2. Checkbox
3. Regular (may be default type in `TestItemDetails`)